app/src/main/java/org/mifos/mobile/api/BaseApiManager.kt

```diff
@@ -89,4 +94,13 @@ class BaseApiManager @Inject constructor(preferencesHelper: PreferencesHelper) {
         createService(preferencesHelper.baseUrl, preferencesHelper.tenant,
                 preferencesHelper.token)
     }
+
+
+    fun getFineractPaymentHubApi(): FineractPaymentHubService? {
```

Comment:
do we need them here, clarify

Reply:
Some APIs of `PaymentHub` have different base url therefore it is necessary to include them here also.